src/Widgets/NavigationWidget/NavigationWidgetClass.ts

```diff
@@ -5,6 +5,7 @@ export const NavigationWidget = provideWidgetClass('NavigationWidget', {
     metaNavigationCart: 'reference',
     metaNavigationObjs: 'referencelist',
     metaNavigationPortalOverview: 'reference',
+    metaNavigationSearchResultsPage: ['reference', { only: 'SearchResults' }],
```

Comment:
After your suggestion I briefly tried out to make a `SearchResultsWidget`. As far as I know this is not possible, because `provideComponent` will only pass `params` to objs, but not to widgets.

Reply:
Wouldn't [Scrivito.currentPageParams](https://docs.scrivito.com/currentpageparams-c4072f845258528c)() do the job?